tiledb/api/src/array/enumeration/strategy.rs

```diff
@@ -0,0 +1,95 @@
+use std::cmp::Ordering;
+
+use proptest::collection::vec;
+use proptest::prelude::*;
+
+use crate::array::EnumerationData;
+use crate::datatype::strategy::*;
+use crate::util::strategy::AnyNumCmp;
+use crate::{fn_typed, Datatype};
+
+pub fn prop_enumeration_name() -> impl Strategy<Value = String> {
+    proptest::string::string_regex("[a-zA-Z0-9_]+")
+        .expect("Error creating enumeration name strategy")
+}
+
+fn prop_cell_val_num() -> impl Strategy<Value = Option<u32>> {
+    Just(None)
+}
+
+fn prop_ordered() -> impl Strategy<Value = bool> {
+    any::<bool>()
+}
+
+fn do_cmp<T: AnyNumCmp>(a: &T, b: &T) -> Ordering {
+    a.cmp(b)
+}
+
+fn prop_enumeration_values(datatype: Datatype) -> BoxedStrategy<Box<[u8]>> {
+    fn_typed!(datatype, DT, {
+        vec(any::<DT>(), 1..1024)
```

Comment:
Minor request to change these to named constants so we can grep more easily for them in a future where we want them to be configurable

Reply:
Done.